cargo-lock/README.md

```diff
@@ -41,7 +41,7 @@ accompanied by a minor version bump.
 This crate provides a `cargo lock` subcommand which can be installed with:
 
 ```text
```

Comment:
Does GitHub still not honor ```console here?

Reply:
Tapping this copy button

<img width="741" alt="Screenshot 2023-05-03 at 12 06 25 PM" src="https://user-images.githubusercontent.com/28851/236005732-9bb00fe3-1afa-4ee8-acc1-341e3da1e4e2.png">

<img width="130" alt="Screenshot 2023-05-03 at 12 08 37 PM" src="https://user-images.githubusercontent.com/28851/236006014-13c78bc1-c15a-4ba7-affb-32f4fae55bb6.png">

copies this text: `$ cargo install cargo-lock --features=cli`

Reply:
That's what ```console was supposed to resolve but I'm uncertain of if GitHub ever implemented it

Reply:
`console` [highlights the text](https://github.com/phatblat/rustsec/blob/4628665726997f6a83aef31536401a1b13c064df/cargo-lock/README.md#command-line-interface) in blue, but restoring the [prompt character](https://github.com/phatblat/rustsec/blob/4980ed327ca45fbc5d081d6398ad83afff69b513/cargo-lock/README.md#command-line-interface) still includes the `$` when copied.This is in

Reply:
Okay, unfortunate